docs/src/main/sphinx/admin/properties-task.rst

```diff
@@ -90,6 +102,16 @@ guaranteed to run in order to prevent deadlocks. A lower value may improve respo
 for new tasks, but can result in underutilized resources. A higher value can increase
 resource utilization, but uses additional memory.
 
+``task.min-drivers-per-task``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** ``integer``
+* **Minimum value:** ``1``
+* **Default Value:** ``3``
+
+The minimum number of drivers guaranteed to run concurrently for a single task given
+there is sufficient work to do. 
```

Comment:
"given there is sufficient work to do" is vague and open to interpretation. Please clarify.

Reply:
Good point. I had a quick read through the source code / other documentation and have updated it to 
```
The minimum number of drivers guaranteed to run concurrently for a single task given
the task has remaining splits to process.
```
Does this sound specific enough?